poetry.lock

```diff
@@ -1131,19 +1131,19 @@ vectorstore-mmr = ["numpy (>=1)", "simsimd (>=3)"]
 
 [[package]]
 name = "exasol-bucketfs"
-version = "2.1.0"
+version = "1.1.0"
```

Comment:
exasol-bucketfs went from 2.1.0 to 1.1.0
exasol-saas-api went from 2.4.0 to 1.1.1

Is this something that we'd want to happen?

It could indicate that some dependency is holding down a required transitive value. This could be problematic in the long-term if we allow up to version 3.0 in exasol-saas-api, but only test at 1.1.1